light_curves/code_src/panstarrs_functions.py

```diff
@@ -26,28 +26,21 @@ def panstarrs_get_lightcurves(sample_table, *, radius=1):
     #read in the panstarrs object table to lsdb
     #this table will be used for cross matching with our sample's ra and decs
     #but does not have light curve information
-    panstarrs_object = lsdb.read_hipscat(
-        's3://stpubdata/panstarrs/ps1/public/hipscat/otmo', 
-        storage_options={'anon': True},
-        columns=[
-            "objID",  # PS1 ID
-            "raMean", "decMean",  # coordinates to use for cross-matching
-            "nStackDetections",  # some other data to use
-        ]
-    )
+    panstarrs_object = lsdb.read_hats(UPath('s3://stpubdata/panstarrs/ps1/public/hats/otmo', anon=True),    margin_cache=UPath('s3://stpubdata/panstarrs/ps1/public/hats/otmo_10arcs', anon=True),  columns=[ "objID",  # PS1 ID
```

Comment:
please cut up this very long line (and the other similar one below)

Reply:
Is there a good resource on how to do this?  I tried cutting these up and ended up with syntax errors.

Reply:
done.